projects/plugins/jetpack/tests/e2e/bin/container-setup.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Exit if any command fails.
+set -e
+
+#####
+# This script is designed to be running inside the WP container
+# it is basically a hacky way to insert arbitrary PHP code into wp-config without messing with bash escaping etc.
+# Also, it creates a debug log file
+
+function usage {
+	echo "usage: $0 command"
+	echo "  wp-setup                     Setup wp-config"
+	echo "  gb-prerelease               Pull latest Guteberg pre-prelease"
+	echo "  -h | usage                   output this message"
+	exit 1
+}
+
+function wp_config {
+	touch wp-content/debug.log
+	chown www-data:www-data wp-content/debug.log
+
+	sed -i "/\/\* That's all, stop editing! Happy publishing. \*\//i\
+	define( 'E2E_REQUEST_URL', ( ! empty( \\\$_SERVER['HTTPS'] ) ? 'https://' : 'http://' ) . ( ! empty( \\\$_SERVER['HTTP_HOST'] ) ? \\\$_SERVER['HTTP_HOST'] : 'localhost' ) );\n\
+	define( 'WP_SITEURL', E2E_REQUEST_URL );\n\
+	define( 'WP_HOME', E2E_REQUEST_URL );\n\
+	" wp-config.php
+}
+
+function gb_prerelease {
+	ZIP_PATH=${2:-"wp-content/gutenberg.zip"}
+
+	rm -rf $ZIP_PATH wp-content/plugins/gutenberg
+	BROWSER_URL=$(curl -s https://api.github.com/repos/Wordpress/gutenberg/releases/latest | grep browser_download_url | cut -d '"' -f 4)
```

Comment:
Not sure if this is intended, but this is currently returning the latest release regardless of pre-release flag (currently prints `https://github.com/WordPress/gutenberg/releases/download/v10.1.1/gutenberg.zip`). 

That being said, running the periodic tests against the `latest` version regardless of its `pre-release` status, might just be what we want. What do you think? I think we might then be able to live with only one (set of?) jobs instead of one for stable and other for pre-release.

Reply:
@brbrr ^

Reply:
Yeah, I miss that. Actually, I'm not sure what is "pre-release" is :) 

What would be the best way to get the correct zip?

Reply:
> We'd have to talk to GH's REST API to get a list of releases, and locate the latest release in it that has the `prerelease` field set to `true`

Alternatively, we can try computing* the release name -- based on the latest stable (i.e. non-pre)release :grimacing: : https://docs.github.com/en/rest/reference/repos#get-a-release-by-tag-name 

*semver-increment `minor`; if >9, set minor to 0, semver-increment major; in both cases, append `-rc.1`.

Reply:
Forgot to say, it might make sense to use an [`actions/github-script`](https://github.com/actions/github-script) based step to write all that logic in JavaScript (with an Octokit-based wrapper for GH's REST API readily available), rather than doing a ton of shell scripting and `curl`ing.

Reply:
Thanks for the suggestion Bernie! I decided to use `github-script`, as it's the most simple and straightforward way to get what I need.

Reply:
> That being said, running the periodic tests against the latest version regardless of its pre-release status, might just be what we want. What do you think? I think we might then be able to live with only one (set of?) jobs instead of one for stable and other for pre-release.

I'd like to point out this idea again, in case you missed it. If we focus on pre-releases only, we'll probably miss testing against the stable release before it gets on WPCOM edge or production, since it won't be in the 12hr testing interval. Perhaps it'd be best to just get the latest release whether it's a `prerelease` or not?

Reply:
@fullofcaffeine yeah, I get confused by the "changing requirements" :) Would you be okay with running the tests only with the latest release?

Reply:
> @fullofcaffeine yeah, I get confused by the "changing requirements" :) Would you be okay with running the tests only with the latest release?

Yeah, I think this could work :) We can always adjust later.